stdpopsim/catalog/homo_sapiens.py

```diff
@@ -354,7 +358,7 @@ def _african():
     populations = [
         stdpopsim.Population(name="AFR", description="African"),
     ]
-    citations = [_tennessen_et_al]
+    citations = [_tennessen_et_al.because(stdpopsim.CiteReason.DEM_MODEL)]
```

Comment:
same as above-- no reason to use this code here if we set the reason originally as with the other citations